builder/azure/common/template/template.go

```diff
@@ -4,8 +4,14 @@
 package template
```

Comment:
Out of curiosity again, what is being templated here? I'm not sure I understand what's the use for this package.